monkelib/API/get_data.py

```diff
@@ -99,47 +102,52 @@ def get_user_repo(username: str) -> bool:
         return False
 
 
-def get_most_starred_python_repos() -> bool:
+def get_most_starred_repos() -> bool:
     """
-    This function gets the most starred Python repositories from the GitHub API.
+    This function gets the most starred repositories for multiple languages from the GitHub API.
     :return: True if the request was successful, False if not.
     """
 
     url: str = 'https://api.github.com/search/repositories'
-    query_params: dict = {
-        'q': 'language:python',
-        'per_page': 10  # Number of results per page (max 100)
-    }
-    # Load the token from the pat.json file
+
+    queryList = []
+    for language in LANGUAGE_LIST:
+        queryList.append({
+            'q': f'language:{language}',
+            'per_page': 75,
+        })
+
```

Comment:
What does this function do exactly? is the per_page: 75 the number of projects per language or what exactly?